Beta docs (#860)
microsoftgraph/entra-powershell: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupAppRoleAssignment.md

```

## Description

The `Get-EntraBetaGroupAppRoleAssignment` cmdlet gets a group application role assignment in Microsoft Entra ID. Specify the `ObjectId` parameter to get a group application role assignment.

## Examples

### Example 1: Retrieve application role assignments of a group

```powershell
Connect-Entra -Scopes 'Directory.Read.All'
$GroupId = (Get-EntraBetaGroup -Top 1).ObjectId
Get-EntraBetaGroupAppRoleAssignment -ObjectId $GroupId
```

```Output
ObjectId                                    ResourceDisplayName                 PrincipalDisplayName
--------                                    -------------------                 --------------------

> MSVrBV4APk--eAGnHqMKBLflsQG3rU1EmDFKvgra41I Microsoft Device Management Checkin Ask HR

Reviewer: Kindly remove this part:

The first command gets the object ID of a group by using the [Get-EntraBetaGroup](./Get-EntraBetaGroup.md) cmdlet.
The command stores the ID in the $GroupId variable.

Author: Changes done

> The second command gets the application role assignments of the group in $GroupId.

Reviewer: ```suggestion
This example retrieves the application role assignments of the group.
```
